src/components/app/EnterpriseAppPageRoutes.jsx

```diff
@@ -12,6 +12,8 @@ import { features } from '../../config';
 import { LicenseActivationPage } from '../license-activation';
 import { PathwayProgressPage } from '../pathway-progress';
 
+// TODO: Techdebt, Create Context wrapper around this page for enterpriseCuration
```

Comment:
nit: Is "this page" is a bit misleading since this `EnterpriseAppPageRoutes` component isn't about a specific page?